### PR TITLE
change naming convetion

### PR DESCRIPTION
**Pull request by amarquesmoura**
this changes files and code to use Locker instead of Spot

### CODE REVIEW

File: client/js/App.js [hunk 1]
```diff
@@ -1,40 +1,42 @@
-import React, {Component} from 'react';
-import {Platform, StyleSheet, Text, View} from 'react-native';
+import React, { Component } from "react";
+import { Platform, StyleSheet, Text, View } from "react-native";
+import RootStackNavigator from "../js/navigation/RootStackNavigator";
 
 const instructions = Platform.select({
```
Comment (siddharthparmar7) [inline client/js/App.js]: you can remove this const. We do not need it.

File: client/js/App.js [hunk 2]
```diff
@@ -1,40 +1,42 @@
...
-      <View style={styles.container}>
-        <Text style={styles.welcome}>Welcome to React Native!</Text>
-        <Text style={styles.instructions}>To get started, edit App.js</Text>
-        <Text style={styles.instructions}>{instructions}</Text>
-      </View>
+      // <View style={styles.container}>
+      //   <Text style={styles.welcome}>Welcome to React Native!</Text>
+      //   <Text style={styles.instructions}>To get started, edit App.js</Text>
+      //   <Text style={styles.instructions}>{instructions}</Text>
+      // </View>
```
Comment (siddharthparmar7) [inline client/js/App.js]: Please remove all the commend code.

File: client/js/App.js [hunk 3]
```diff
@@ -1,40 +1,42 @@
...
+      <RootStackNavigator />
     );
   }
 }
 
 const styles = StyleSheet.create({
```
Comment (siddharthparmar7) [inline client/js/App.js]: Please remove these styles.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the changes.

**Comment by amarquesmoura** [comment]
Done.

**Review by siddharthparmar7 — APPROVED** [review]
Nice work!

**Pull request merged**